server/backup_runner.go

```diff
@@ -70,6 +74,82 @@ func (r *BackupRunner) Run(ctx context.Context, wg *sync.WaitGroup) {
 	}
 }
 
+// TODO(dragonly): Make best effort to assure that users could recover to at least RetentionPeriodTs ago.
```

Comment:
This isn't enforcing policy. We are doing purging backups.

Reply:
SG.